src/app/dataset-view/additional-components/dataset-settings-component/tabs/variables-and-secrets/dataset-settings-secrets-manager-tab.component.html

```diff
@@ -1,12 +1,19 @@
 <div class="w-75 mb-100">
     <h2>Variables and secrets</h2>
     <mat-divider class="mt-1"></mat-divider>
-    <p class="text-muted mt-2 mb-2">Key-value pairs associated with dataset</p>
+    <p class="text-muted mt-2 mb-2">
+        Variables and secrets allow you to provide configuration parameters to ingest flows. Variables can be used for
+        non-sensitive values like page limits while secrets can be used for sensitive values like API keys and auth
+        tokens. Learn more about ingest parametrization
+        <a target="_blank" href="https://docs.kamu.dev/cli/ingest/input-formats/?highlight=env#dealing-with-api-keys"
+            >here </a
```

Comment:
would this underline a blank space after "here"?

Reply:
No, it won't. The space is added by the linter.